test/evergreen.yml

```diff
@@ -2830,6 +2830,9 @@ buildvariants:
       --enable-strict --enable-static --prefix=$(pwd)/LOCAL_INSTALL
     test_env_vars:
       PATH=/opt/mongodbtoolchain/v3/bin:$PATH
+    upload_source_dir: mongo-tests/largescale/many-collection/dbpath/diagnostic.data
+    upload_dest_dir: many-coll
+    upload_filename: diagnostic.data.tgz
```

Comment:
Please note expansions apply to all tasks in this build variant. Hope these expansion values make sense for both tasks currently included in this build variant.